test/test_quantization.py

```diff
@@ -311,6 +311,41 @@ def checkQuantized(model):
 
         checkQuantized(model)
 
+    @given(qengine=st.sampled_from(("qnnpack", "fbgemm")))
```

Comment:
NIT: Can we have a better name than train_save_load_eval? There is no training here.